drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -116,6 +116,7 @@
                 <h3>{% trans "Items" %}</h3>
             </header>
             <section class="tab-content">
+                <div class="xblock-editor-error-messages" aria-live="polite" tabindex="0"></div>
```

Comment:
That feels very icky-- only showing it on one tab because that's where it is currently used. The JavaScript method that you call to show the notification is generic, correct? Isn't there a general place where it can be shown (perhaps above the "Items" label)? It felt weird anyway that the error was injected into that location anyway. It's great if the error is inline with the control that caused it, but that isn't the case; it was just put into a place that felt a bit arbitrary anyway.